tests/spread/integration/apache2-bin/task.yaml

```diff
@@ -0,0 +1,49 @@
+summary: Integration tests for apache2-bin
+
+execute: |
+  rootfs="$(install-slices apache2-bin_bins base-passwd_data)"
+
+  chroot "${rootfs}/" apache2 -v
```

Comment:
I would suggest putting a blank empty line after this smoke test to better separate it visually from all the setup below for the functional test.